### PR TITLE
Lesson 01 Assignment - Course Project

### PR DESCRIPTION
**Pull request by mayele23**
Greetings,
I have created multiple pages of the mockup as well as updated the basics html codes for it. I have also put all the images in the images folder, i found it easy to work that way. There are also a couple of pics missing, i just duplicated them for the first step of the course. I have not adjusted or styled any of the images i have uploaded. I have not uploaded the hero image because i want to do it using css. I find it very easy to set it as a background image. If its a must you see it now, i will make the update whenever told. Thank You! Have a blessed day.

### CODE REVIEW

File: adopt-a-dog/blog.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (jimmynono) [inline adopt-a-dog/blog.html]: You'll want to delete these extra lines.

File: adopt-a-dog/blog.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  <section class="hero">
+    <div class="hero-inner">
+      <h1>Seattle's premiere dog adoption agency</h1>
+    </div>
+  </section>
+  <header class="masthead">
+    <div class="container">
+      <nav class="main-nav">
+        <ul class="menu">
+          <li><a href="#">Home</a></li>
```
Comment (jimmynono) [inline adopt-a-dog/blog.html]: Make sure these links work.

File: adopt-a-dog/blog.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          <li><a href="#">Dogs</a></li>
+          <li><a href="#">Blog</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section class="blog-container">
+    <div class="blog-1">
```
Comment (jimmynono) [inline adopt-a-dog/blog.html]: You'll want to share a single class with all of the blog posts for easy styling

File: adopt-a-dog/blog.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+        nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+        reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+        culpa qui officia deserunt mollit anim id est laborum.
+      </p>
+    </div>
+
+    <div class="blog-2">
```
Comment (jimmynono) [inline adopt-a-dog/blog.html]: Nice work on grouping these

File: adopt-a-dog/index.html [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+  </section>
+
+    <section class="main-container-2-inner">
+      <div class="dog-1">
+        <img src="images/alex-lacayo-756508-unsplash.jpg" alt="eric ward hugging a dog">
+        <h3>Mruphy</h3>
+        <h4>Cost to Adopt: $123.45</h4>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+          sed do eiusmod tempor incididunt</p>
+        <button aria-label="Adopt" type="Submit" class="button-1">Adopt</button>
```
Comment (jimmynono) [inline adopt-a-dog/index.html]: aria-lable! Love it!

File: adopt-a-dog/index.html [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+      </div>
+
+      <div class="dog-2">
```
Comment (jimmynono) [inline adopt-a-dog/index.html]: same with the dogs, it's good to have a single class for all of them

**Comment by mayele23** [comment]
Thank you for comments!
My code has been updated accordingly from the last time we reviewed it in
class together. Things are looking much cleaner and better for the next
round.

Thank you for the feedback.

**Pull request closed without merging**